runelite-client/src/main/java/net/runelite/client/plugins/mta/MTAConfig.java

```diff
@@ -81,18 +88,57 @@ default boolean alchemy()
 	@ConfigItem(
 			keyName = "enchantmentEnable",
 			name = "Enable",
-			description = "Configures whether or not the enchantment room overlay is enabled.",
+			description = "Configures whether or not the Enchanting Chamber overlay is enabled.",
+			position = 0,
 			section = enchantmentGroup
 	)
 	default boolean enchantment()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "highlightDragonstones",
+			name = "Highlight dragonstones",
+			description = "Enable markers indicating spawned dragonstones.",
+			position = 1,
+			section = enchantmentGroup
+	)
+	default boolean highlightDragonstones()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "dragonstoneMode",
+			name = "Highlighting mode",
+			description = "Chooses which dragonstones should be highlighted.",
+			position = 2,
+			section = enchantmentGroup
+	)
+	default MarkingMode dragonstoneMode()
+	{
+		return MarkingMode.ALL;
+	}
+
+	@ConfigItem(
+			keyName = "defaultColor",
+			name = "Highlight color",
+			description = "The highlight color for dragonstones.",
+			position = 3,
+			section = enchantmentGroup
+	)
+	default Color dragonstoneColor()
+	{
+		return Color.YELLOW;
+	}
+
+
 	@ConfigItem(
 		keyName = "graveyardEnable",
 		name = "Enable",
-		description = "Configures whether or not the graveyard room overlay is enabled.",
+		description = "Configures whether or not the Creature Graveyard overlay is enabled.",
+		position = 0,
```

Comment:
Items which are alone in their config sections do not need positions assigned to them.

Reply:
Removed with latest commit.